CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
+-   Environment variable can be used in the compose file. See
+    https://github.com/docker/compose/blob/master/docs/yml.md#variable-substitution
+-   Multiple compose files can be specified, allowing you to override
+    setting in the default compose file. See
+    https://github.com/docker/compose/blob/master/docs/reference/docker-compose.md
+    for more details.
+-   Configuration validation is now a lot more strict
+-   `up` now waits for all services to exit before shutting down
+-   Support for the new docker networking can be enabled with
+    the `--x-networking` flag
+
+New Features
+
+-   Compose can be installed and run as a docker image
```

Comment:
It might be worth sticking this at the bottom, clarifying that it's an experimental feature - it might confuse users if they think this is the default. (How does this work? Do I have to change what I'm doing? etc)